src/components/pages/ActionListPage.tsx

```diff
@@ -85,51 +121,20 @@ const SortButtons = styled(ToggleButtonGroup)`
   }
 `;
 
-const ActionsViewTabs = styled.div`
-  background-color: ${(props) => props.theme.brandDark};
-  margin-bottom: ${(props) => props.theme.spaces.s400};
-`;
-
-const Tab = styled.button`
-  background: ${(props) => props.theme.brandDark};
-  color: ${(props) => props.theme.themeColors.white};
-  display: inline-block;
-  border: none;
-  margin: 0;
-  padding: ${(props) =>
-    `${props.theme.spaces.s050} ${props.theme.spaces.s150} ${props.theme.spaces.s100}`};
-  text-decoration: none;
-  cursor: pointer;
-  text-align: center;
-
-  &:hover,
-  &:focus {
-    color: ${(props) => props.theme.brandLight};
-  }
-  &.active {
-    color: ${(props) => props.theme.graphColors.blue070};
-    background: ${(props) => props.theme.graphColors.grey030};
-    &:hover {
-      color: ${(props) => props.theme.themeColors.black};
-    }
-  }
-
-  .icon {
-    width: 1.5rem !important;
-    height: 1.5rem !important;
-    vertical-align: middle;
-  }
+const ViewSelectorBar = styled.div`
+  margin-top: 1rem;
+  margin-bottom: 1rem;
 `;
 
 const getSortOptions = (
   t: TFunction,
   hasEfficiency: boolean,
   showAccumulatedEffects: boolean
 ): SortActionsConfig[] => [
-  {
-    key: 'STANDARD',
-    label: t('actions-sort-default'),
-  },
+  { 
+    key: 'STANDARD', 
+    label: t('actions-sort-default')
+   },
```

Comment:
Is this adding white space after the curly braces? Perhaps your editor has added these? Try running `pnpm run prettier:fix`